src/main/java/Astra.java

```diff
@@ -0,0 +1,101 @@
+import java.util.Scanner;
+import java.util.ArrayList;
+
+class Task {
```

Comment:
I feel that the classes can be placed in separate files for clarity :))
```suggestion
class Task {
```